client-app/pages/loyalty/loyalty.vue

```diff
@@ -0,0 +1,456 @@
+                            @click="openEditMode()"
+                            v-t="'pages.loyalty.points_operations.operations_list.add_new_operation_button'"
+                          ></VcButton>
+                        </div>
+                      </div>
+                    </td>
+                  </tr>
+                </template>
+                <template #desktop-skeleton>
+                  <tr v-for="i of itemsPerPage" :key="i" class="even:bg-gray-50">
+                    <td class="p-5">
+                      <div class="h-5 bg-gray-200 animate-pulse"></div>
+                    </td>
+                    <td class="w-1/2 p-5">
+                      <div class="h-5 bg-gray-200 animate-pulse"></div>
+                    </td>
+                    <td class="p-5">
+                      <div class="h-5 bg-gray-200 animate-pulse"></div>
+                    </td>
+                    <td class="p-5">
+                      <div class="h-5 bg-gray-200 animate-pulse"></div>
+                    </td>
+                  </tr>
+                </template>
+              </VcTable>
+
+              <VcInfinityScrollLoader
+                class="my-6"
+                v-if="infiniteScrollEnabled && !pointsOperationsLoading"
+                :loading="morePointsOperationsLoading"
+                distance="400"
+                @visible="loadMoreOperations"
+              ></VcInfinityScrollLoader>
+              <VcScrollTopButton v-if="infiniteScrollEnabled"></VcScrollTopButton>
+            </template>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import {
+  ITableColumn,
+  VcButton,
+  VcCard,
+  VcCheckbox,
+  VcInfinityScrollLoader,
+  VcScrollTopButton,
+  VcTable,
+} from "@/components";
+import { AccountNavigation } from "@/shared/account";
+import { PointsOperationForm, usePointsOperations, useUserBalance } from "@/shared/loyalty";
+import { computed, ComputedRef, onMounted, Ref, ref } from "vue";
+import { clone } from "lodash";
+import { PointsOperationsSearchParams } from "@/core/api/graphql/loyalty";
+import { InputRegisterPointsOperationType } from "@/core/api/graphql/types";
+import { currentUserId, storeId } from "@/core/constants";
+import { breakpointsTailwind, useBreakpoints } from "@vueuse/core";
+import { useI18n } from "vue-i18n";
+
+const { t } = useI18n();
+
+const breakpoints = useBreakpoints(breakpointsTailwind);
+const {
+  loading: pointsOperationsLoading,
+  loadingMore: morePointsOperationsLoading,
+  pointsOperations,
+  total,
+  pages,
+  fetchMyPointsOperations,
+  fetchMorePointsOperations,
+  addPointsOperation,
+} = usePointsOperations({ userId: currentUserId, storeId });
+
+const {
+  loading: userBalanceLoading,
+  myUserBalance,
+  loadMyUserBalance,
+} = useUserBalance({ userId: currentUserId, storeId });
+
+const isMobile = breakpoints.smaller("md");
+const editingMode: Ref<boolean> = ref(false);
```

Comment:
It's not needed to explicitly pass such a type, simple types are determined automatically (like on lines 327 and 328).